tools/output/blocks.js

```diff
@@ -1090,5 +1090,10 @@ module.exports = [
     {
         'name': 'InYijing_Hexagram_Symbols',
         'bmp': '\u4DC0-\u4DFF'
+    },
+    {
+        'name': 'Inundefined',
```

Comment:
This seems like a bug

Reply:
Good catch! It looks like `unicode-10.0.0` has an "undefined" block, so I'll open an issue in node-unicode-data.